Resources/Prototypes/_NF/Entities/Clothing/Head/hardsuit-helmets.yml

```
@@ -161,8 +159,8 @@
   parent: ClothingHeadHardsuitWithLightBaseNfsd
   id: ClothingHeadHelmetHardsuitNfsdExperimental
   categories: [ HideSpawnMenu ]
-  name: experimental nfsd hardsuit helmet
-  description: experimental nfsd hardsuit helmet
+  name: NFSD experimental combat hardsuit helmet
```

```suggestion
  name: NFSD experimental combat suit helmet
```
You dropped the hard.